app/javascript/src/lib/dayjs.js

```diff
@@ -0,0 +1,6 @@
+import dayjs from "dayjs";
```

Comment:
This file should be imported at the root/starting component of your app like say App.jsx. That import statement will ensure dayjs gets extended.